src/about/index.md

```diff
@@ -22,12 +24,15 @@ few features that make Void unique:
    dedicated to the security, quality, and maintainability of this critical
    library.
 - [runit](../config/services/index.md) is used for
-   [init(8)](https://man.voidlinux.org/init.8). This allows Void to support musl
-   as a second libc choice, which would not be possible with
-   [systemd](https://www.freedesktop.org/wiki/Software/systemd/). A side effect
-   of this decision is an init system with clean and efficient operation, and a
-   small code base.
+   [init(8)](https://man.voidlinux.org/init.8) and service supervision. This
+   allows Void to support musl as a second libc choice, which would not be
+   possible with [systemd](https://www.freedesktop.org/wiki/Software/systemd/).
+   A side effect of this decision is an init and service supervision system with
```

Comment:
init and service supervision seems redundant here? i can't think of a great alternative at this moment.

Reply:
i've added this due to a discussion with someone who thought that runit was only used for init, rather than service supervision. (i guess this is what happens when the arguments around systemd get framed as being a debate about init systems.)

Reply:
My comment only addresses the duplication of "init" and "service supervision system". I wonder if the second mention could just say "core system with [...]"? I'm not sure just a guess.

Reply:
oic. Sure, PR updated.